collective/honeypot/configure.zcml

```diff
@@ -1,23 +1,11 @@
 <configure
     xmlns="http://namespaces.zope.org/zope"
     xmlns:browser="http://namespaces.zope.org/browser"
-    xmlns:cmf="http://namespaces.zope.org/cmf"
-    xmlns:five="http://namespaces.zope.org/five"
-    xmlns:i18n="http://namespaces.zope.org/i18n"
     xmlns:zcml="http://namespaces.zope.org/zcml"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
     i18n_domain="collective.honeypot">
 
-  <!-- Register package so we can use the Extensions directory. -->
-  <five:registerPackage package="." />
-
-  <!-- File System Directory Views registration -->
-  <cmf:registerDirectory name="honeypot_oldcomments" />
-  <cmf:registerDirectory name="honeypot_plone3" />
-  <cmf:registerDirectory name="honeypot_plone4" />
-  <cmf:registerDirectory name="honeypot_quinta" />
-
-  <!-- Optional GenericSetup profiles. -->
-  <include file="profiles.zcml" />
+  <include file="fixes.zcml" />
```

Comment:
Is there a reason why ``fixes.zcml`` wasn't automatically loaded? I included it here so that the package is active right away. @mauritsvanrees

Reply:
That is somewhat explained in the [Readme](https://github.com/collective/collective.honeypot/blob/1.x/README.rst#activation-on-plone-3) in the part about activation. On Plone 3, you could either activate the add-on in the quick installer, or activate it by loading `fixes.zcml`, with the downside that this requires `z3c.jbot`, which is not always wanted, and for which the readme warns about possible conflicts.
The next paragraph about Plone 4 says you should simply load `fixes.zcml`.

Anyway, for Plone 5 loading it automatically is definitely the way to go.